test/development/acceptance-app/dynamic-metadata-error.test.ts

```diff
@@ -33,7 +33,7 @@ describe('dynamic metadata error', () => {
       return new ImageResponse(<div>icon</div>)
     }
     `
-    const { cleanup } = await sandbox(
+    await using _sandbox = await createSandbox(
```

Comment:
do we need the underscore in these tests? seems like no conflicts

Reply:
The return value of `createSandbox` is not used, this is to make TypeScript happy. Without `using` keyword, we can just do `await createSandbox(...)`, but you have to assign it to a variable with `using.`